### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by kachkaev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,14 +25,18 @@
       "suppress-exit-code prettier --write"
     ]
   },
+  "resolutions": {
+    "markdown-it": "^12.3.2",
+    "simple-get": "^4.0.1"
```
Comment (kachkaev) [inline package.json:30]: https://github.com/advisories/GHSA-wpg7-2c88-r8xv

File: package.json [hunk 2]
```diff
@@ -25,14 +25,18 @@
       "suppress-exit-code prettier --write"
     ]
   },
+  "resolutions": {
+    "markdown-it": "^12.3.2",
```
Comment (kachkaev) [inline package.json:29]: https://github.com/advisories/GHSA-6vfc-qv3f-vr6c

File: package.json [hunk 3]
```diff
@@ -25,14 +25,18 @@
...
   "dependencies": {
     "@auth0/nextjs-auth0": "^1.7.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-brands-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
-    "@fortawesome/react-fontawesome": "^0.1.16",
+    "@fortawesome/react-fontawesome": "^0.1.17",
     "@prisma/client": "^3.8.1",
-    "@sentry/nextjs": "^6.16.1",
+    "@sentry/nextjs": "^6.17.3",
```
Comment (kachkaev) [inline package.json:39]: https://github.com/getsentry/sentry-javascript/pull/4467

**Review by PiRomant — APPROVED** [review]
На карте ничего не изменилось

**Review by sergous — APPROVED** [review]

**Pull request merged**